profile_library/lifx/LFXA211600RGBW/model.json

```diff
@@ -0,0 +1,17 @@
+{
+  "calculation_strategy": "lut",
+  "created_at": "2024-10-07T13:41:48.240238",
+  "measure_description": "Measured with utils/measure script",
+  "measure_device": "Shelly Plug S",
+  "measure_method": "script",
+  "measure_settings": {
+    "SAMPLE_COUNT": 1,
+    "SLEEP_TIME": 3,
+    "VERSION": "v1.14.10:docker"
+  },
+  "name": "LIFX A21",
+  "standby_power": 0.1,
```

Comment:
One final remark. Could you verify this one?
This seems too low for LIFX lights.
When you are unable to measure because it's too low for Shelly Plug S we can guesstimate, but 0.1 can't be correct imo.